packages/webviz-core/src/panels/Plot/index.stories.js

```diff
@@ -151,7 +151,6 @@ const fixture = {
   activeData: {
     startTime: { sec: 0, nsec: 202050 },
     endTime: { sec: 24, nsec: 999997069 },
-    currentTime: { sec: 0, nsec: 750000000 },
```

Comment:
Why this change?

Reply:
I think this accidentally slipped in, will revert